### DISCUSSION TITLE
Suspected Permission Error Trying To Migrate Files Location On Docker Host

### DISCUSSION THREAD

**Discussion by downj05** [pufferpanel/pufferpanel#1339] [Q&A]
Wanted to move my /var/lib/pufferpanel to a mount with more space, made the mistake of not reading the docs which tells you NOT to do this kind of thing. I have done this, getting (what i suspect are permission errors) trying to start servers (Error: Unable to access jarfile paper.jar) etc when trying to start a Minecraft server. Am I cooked? Is there anyway to fix this? There doesn't seem to be a pufferpanel user/group in the container or on my system I can chown with.

**Comment by LordRalex** [comment]
You need to make your mount as /var/lib/pufferpanel for this to work really.

So, best approach:
Move /var/lib/pufferpanel to a temp location
Mount your new "drive" to /var/lib/pufferpanel
Move files back to this path

then, since you are using Docker, chown the files to root, since that's the user id that ends up being used in Docker.
